pkg/controllers/framework/interface.go

```diff
@@ -37,6 +38,8 @@ type ControllerOption struct {
 	InheritOwnerAnnotations bool
 	WorkerThreadsForPG      uint32
 	WorkerThreadsForGC      uint32
+
+	Config *rest.Config
```

Comment:
Why do we need this change?

Reply:
Do you have any plans or what the custom client needs?

Reply:
I think we add some comments to describe why is this needed and what does it aim to do is better: ) @liuyuanchun11

Reply:
Sorry for not explaining clearly. I am maintaining a project based on the volcano community version, which contains some custom resource controllers and plug-ins, etc. Now I need to add a custom resource clientset, similar to VolcanoClient. The clientset of this custom resource is obviously not suitable to be placed in the ControllerOption structure, so I want to put rest.Config in the ControllerOption, so that it can be passed to my new controller, and then call NewForConfigOrDie(config of the custom resource in this controller ) to generate clientset

Reply:
Will the maintained features be fed back to the community? IMO, there is no need to introduce unused variables.
Just my opinion.If it is an overall feature, it feels more reasonable to introduce this variable